README.md

```diff
@@ -2,67 +2,96 @@
 
-OpenROAD is an integrated chip physical design tool that takes a
-design from synthesized Verilog to routed layout.
-
-An outline of steps used to build a chip using OpenROAD is shown below:
-
-* Initialize floorplan - define the chip size and cell rows
-* Place pins (for designs without pads )
-* Place macro cells (RAMs, embedded macros)
-* Insert substrate tap cells
-* Insert power distribution network
-* Macro Placement of macro cells
-* Global placement of standard cells
-* Repair max slew, max capacitance, and max fanout violations and long wires
-* Clock tree synthesis
-* Optimize setup/hold timing
-* Insert fill cells
-* Global routing (route guides for detailed routing)
-* Antenna repair
-* Detailed routing
-* Parasitic extraction
-* Static timing analysis
-
-OpenROAD uses the OpenDB database and OpenSTA for static timing analysis.
+## About OpenROAD
+
+OpenROAD is the leading open-source, foundational application for
+semiconductor digital design. The OpenROAD flow delivers an
+Autonomous, No-Human-In-Loop (NHIL) flow, 24 hour turnaround from
+RTL-GDSII for rapid design exploration and physical design implementation.
+
+![rtl2gds.webp](./docs/images/rtl2gds.webp)
 
 Documentation is also available [here](https://openroad.readthedocs.io/en/latest/main/README.html).
 
-## Getting Started with OpenROAD
+## OpenROAD Mission
 
-[OpenROAD](https://theopenroadproject.org/) is the leading
-open-source, foundational application for semiconductor digital design.
-It eliminates the barriers of cost, risk and uncertainty in hardware
-design to foster open access, expertise, rapid innovation, and faster
-design turnaround. The OpenROAD application enables flexible flow
+[OpenROAD](https://theopenroadproject.org/) eliminates the barriers
+of cost, schedule risk and uncertainty in hardware design to promote
+open access to rapid, low-cost IC design software and expertise and
+system innovation. The OpenROAD application enables flexible flow
 control through an API with bindings in Tcl and Python.
 
-OpenROAD is a foundational building block in open-source digital flows
-such as [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts),
+OpenROAD is used in both open-source and commercial IC design
+flows such as native [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts),
 [OpenLane](https://github.com/The-OpenROAD-Project/OpenLane) from
 [Efabless](https://efabless.com/), [Silicon Compiler](https://github.com/siliconcompiler/siliconcompiler)
 from Zero ASIC, [Hammer](https://docs.hammer-eda.org/en/latest/Examples/openroad-nangate45.html)
 from Berkeley and [OpenFASoC](https://github.com/idea-fasoc/OpenFASOC) for
 mixed-signal design flows.
```

Comment:
Turning these into bullet points would be more readable.

Reply:
updated